src/views/GovernanceTaskList/TaskListItem.tsx

```diff
@@ -45,6 +46,10 @@ const TaskListItem = ({
         </p>
 
         {status === 'START' && <Button to={link}>Start</Button>}
+        {status === 'CONTINUE' && <Button to={link}>Continue</Button>}
```

Comment:
Ideally we should have `aria-label`'s for these buttons/links. `View Submitted Request Form` is good and has enough context for assistive technology.

But `Start` and `Continue` alone don't have enough context.

I'm not 100% sure how we will accomplish this. This might call for a separate JIRA ticket because:

1. Our `Button` currently doesn't support aria attributes.
2. We need to figure out what the aria text needs to be.
3. This is related to 2. We need to figure out the verbiage and how we pass it to the component. For example, we can easily do `Start Fill in the request form` or `Continue Prepare your Business Case`, but the verbiage isn't correct.

Reply:
Design is looking into this, but it should not block this work